src/handlers/gestures/customGesture.ts

```diff
@@ -0,0 +1,11 @@
+import { ContinousBaseGesture } from './gesture';
+
+export class CustomGesture extends ContinousBaseGesture<Record<string, never>> {
```

Comment:
What is the difference between using Continous vs Discrete here? Does it mean we can't make a discrete handler with "CustomGesture" class?

Reply:
You can make discrete gesture, you just need to control the states correctly. Making it inherit after `ContinousBaseGesture` allows for setting `onUpdate` callback.